### PR TITLE
Fix URL, update refpolicy patches and dependencies

### PR DESCRIPTION
**Pull request by jjmcdn**
* audit_2.7.6.bb : fix error [gzip: stdin: not in gzip format] and checksum
* refpolicy-minimum_git.bb : fix [Failed to resolve typeattributeset statement], dependency for "fsadm" in init.pp
* refpolicy-targeted_2.20170204.bb : added version dependent patches
* patches : separate patches for release 2.20170204 version and 2.20170805+git version

Signed-off-by: Sajjad Ahmed <sajjad_ahmed@mentor.com>
Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>

### CODE REVIEW

**Review by 0xsajjad — APPROVED** [review]
I think in commit description, audit URL related statement is no longer required.

File: recipes-security/refpolicy/refpolicy-git/poky-fc-update-alternatives_bash.patch [hunk 1]
```diff
@@ -1,24 +1,12 @@
...
-+++ refpolicy/policy/modules/kernel/corecommands.fc
-@@ -6,6 +6,7 @@
- /bin/d?ash			--	gen_context(system_u:object_r:shell_exec_t,s0)
- /bin/bash			--	gen_context(system_u:object_r:shell_exec_t,s0)
- /bin/bash2			--	gen_context(system_u:object_r:shell_exec_t,s0)
-+/bin/bash\.bash			--	gen_context(system_u:object_r:shell_exec_t,s0)
- /bin/fish			--	gen_context(system_u:object_r:shell_exec_t,s0)
- /bin/ksh.*			--	gen_context(system_u:object_r:shell_exec_t,s0)
- /bin/mksh			--	gen_context(system_u:object_r:shell_exec_t,s0)
+diff --git a/policy/modules/kernel/corecommands.fc b/policy/modules/kernel/corecommands.fc
```
Comment (0xsajjad) [inline recipes-security/refpolicy/refpolicy-git/poky-fc-update-alternatives_bash.patch:1]: non-standard patch format, subject missing

File: recipes-security/refpolicy/refpolicy-git/poky-policy-add-rules-for-var-log-symlink-apache.patch [hunk 2]
```diff
@@ -1,31 +1,12 @@
...
-
-Upstream-Status: Inappropriate [only for Poky]
-
-Signed-off-by: Xin Ouyang <Xin.Ouyang@windriver.com>
-Signed-off-by: Joe MacDonald <joe_macdonald@mentor.com>
----
- policy/modules/contrib/apache.te |    1 +
- 1 file changed, 1 insertion(+)
-
+diff --git a/policy/modules/contrib/apache.te b/policy/modules/contrib/apache.te
```
Comment (0xsajjad) [inline recipes-security/refpolicy/refpolicy-git/poky-policy-add-rules-for-var-log-symlink-apache.patch:1]: same as above. the format needs to be fixed

**Comment by 0xsajjad** [comment]
@joeythesaint tell me if we need to fix these changes, I'll commit the updated fix version.

**Comment by jjmcdn** [comment]
@sajjad238 Yeah, if we can make these changes it would be good.  I am quite tight on time right now so if you've got the time to do that, it would be great, but otherwise I'll try to circle back to it in the next day or so.

**Comment by 0xsajjad** [comment]
@joeythesaint here is the fixed version, take a look
https://github.com/sajjad238/meta-selinux-updated-fix/commit/7afa05831684642de5def2bf7108ff6f6697b29e

**Pull request merged**